packages/core-mobile/app/new/features/receive/screens/ReceiveScreen.tsx

```diff
@@ -29,21 +29,15 @@ export const ReceiveScreen = (): ReactNode => {
       case NetworkVMType.BITCOIN:
         return activeAccount?.addressBTC ?? ''
       case NetworkVMType.AVM:
-        return activeAccount?.addressAVM ?? ''
+        return activeAccount?.addressAVM.split('-')[1] ?? ''
       case NetworkVMType.PVM:
-        return activeAccount?.addressPVM ?? ''
+        return activeAccount?.addressPVM.split('-')[1] ?? ''
       case NetworkVMType.EVM:
       default:
         return activeAccount?.addressC ?? ''
     }
   }, [activeAccount, selectedNetwork])
 
-  const qrCodeAddress = useMemo(() => {
-    if (selectedNetwork.chainId && isXPChain(selectedNetwork.chainId))
-      return address.split('-')[1]
-    return address
-  }, [address, selectedNetwork])
-
```

Comment:
make sure we only render 1 row for x/p address
<img src="https://github.com/user-attachments/assets/036f5895-154f-48e3-9415-f088b90139bc" width="300"/>